### ISSUE TITLE
Fix docs.rs builds

### ISSUE THREAD

**Issue by jcape** [mobilecoinfoundation/sgx#89]
[Example failure](https://docs.rs/crate/mc-sgx-dcap-sys-types/0.1.0/builds/613724):

```
# rustc version
rustc 1.65.0-nightly (9c20b2a8c 2022-08-17)
# docs.rs version
docsrs 0.6.0 (24bf3b2 2022-08-08)

# build log
[INFO] running `Command { std: "docker" "create" "-v" "/home/cratesfyi/workspace/builds/mc-sgx-dcap-sys-types-0.1.0/target:/opt/rustwide/target:rw,Z" "-v" "/home/cratesfyi/workspace/builds/mc-sgx-dcap-sys-types-0.1.0/source:/opt/rustwide/workdir:ro,Z" "-v" "/home/cratesfyi/workspace/cargo-home:/opt/rustwide/cargo-home:ro,Z" "-v" "/home/cratesfyi/workspace/rustup-home:/opt/rustwide/rustup-home:ro,Z" "-e" "SOURCE_DIR=/opt/rustwide/workdir" "-e" "CARGO_TARGET_DIR=/opt/rustwide/target" "-e" "DOCS_RS=1" "-e" "CARGO_HOME=/opt/rustwide/cargo-home" "-e" "RUSTUP_HOME=/opt/rustwide/rustup-home" "-w" "/opt/rustwide/workdir" "-m" "3221225472" "--cpus" "3" "--user" "1001:1001" "--network" "none" "ghcr.io/rust-lang/crates-build-env/linux@sha256:824c28ce115f6b999260af6986f3384c158e782489540e41c0b698ca1d9fd927" "/opt/rustwide/cargo-home/bin/cargo" "+nightly" "rustdoc" "--lib" "-Zrustdoc-map" "-Z" "unstable-options" "--config" "build.rustdocflags=[\"-Z\", \"unstable-options\", \"--emit=invocation-specific\", \"--resource-suffix\", \"-20220817-1.65.0-nightly-9c20b2a8c\", \"--static-root-path\", \"/\", \"--cap-lints\", \"warn\", \"--disable-per-crate-search\", \"--extern-html-root-takes-precedence\"]" "-Zunstable-options" "--config=doc.extern-map.registries.crates-io=\"https://docs.rs/{pkg_name}/{version}/x86_64-unknown-linux-gnu\"" "-j3" "--target" "x86_64-unknown-linux-gnu", kill_on_drop: false }`
[INFO] [stdout] acceff33858ddf2b1892b45cb2d8bb02840772e9c2fc1c6c20ad28403a719b98
[INFO] [stderr] WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
[INFO] running `Command { std: "docker" "start" "-a" "acceff33858ddf2b1892b45cb2d8bb02840772e9c2fc1c6c20ad28403a719b98", kill_on_drop: false }`
[INFO] [stderr]    Compiling mc-sgx-dcap-sys-types v0.1.0 (/opt/rustwide/workdir)
[INFO] [stderr] error: failed to run custom build command for `mc-sgx-dcap-sys-types v0.1.0 (/opt/rustwide/workdir)`
[INFO] [stderr] 
[INFO] [stderr] Caused by:
[INFO] [stderr]   process didn't exit successfully: `/opt/rustwide/target/debug/build/mc-sgx-dcap-sys-types-3d1c7f4f994780db/build-script-build` (exit status: 101)
[INFO] [stderr]   --- stderr
[INFO] [stderr]   wrapper.h:1:10: fatal error: 'sgx_ql_quote.h' file not found
[INFO] [stderr]   thread 'main' panicked at 'Unable to generate bindings: ClangDiagnostic("wrapper.h:1:10: fatal error: 'sgx_ql_quote.h' file not found\n")', build.rs:50:10
[INFO] [stderr]   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[INFO] running `Command { std: "docker" "inspect" "acceff33858ddf2b1892b45cb2d8bb02840772e9c2fc1c6c20ad28403a719b98", kill_on_drop: false }`
[INFO] running `Command { std: "docker" "rm" "-f" "acceff33858ddf2b1892b45cb2d8bb02840772e9c2fc1c6c20ad28403a719b98", kill_on_drop: false }`
[INFO] [stdout] acceff33858ddf2b1892b45cb2d8bb02840772e9c2fc1c6c20ad28403a719b98
```

**Comment by jcape** [comment]
I *think* this has been fixed since the release, we'll see with 0.2...

**Comment by jcape** [comment]
Next step is to disable `sgxsdk` action in docs CI.

**Comment by jcape** [comment]
Fixed in 0.2.1
